src/index.scss

```diff
@@ -1,11 +1,47 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter&family=Philosopher&display=swap');
+
 :root {
   font-size: 16px;
+
+  // Font Variables
+  --inter: 'Inter', sans-serif;
+  --philosopher: 'Philosopher', sans-serif;
```

Comment:
Do we not need any global styles where we use the philosopher font style?

Reply:
only place I saw philosopher font is used in the logo.

Reply:
Ah okay, then it's all good :)